modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js

```diff
@@ -152,7 +153,8 @@ const DocumentLibrary = ({
 				<ClayInput.Group>
 					<ClayInput.GroupItem prepend>
 						<ClayInput
-							aria-label={Liferay.Language.get('file')}
+							aria-label="upload"
```

Comment:
@rijema, will the value of the property `aria-label` be used by a screen reader? If so, then we should consider the language to set the proper attribute's value.

Reply:
Yes, the value will be used by the ScreenReader. To explain what was my thought: I looked for other aria-label on others fields, and I found aria-label="text" and aria-label="numeric". So, in that way, I tried to follow a pattern

I added a new commit making the change to use the properly label value. Let me know if you want me to update this pull request with a new commit or if I have to squash in some of these.